ddev/ddev
# feat: invoke dbeaver inside wsl2, fixes #5375

DBeaver support inside wsl2

## The Issue
We are not able to call `ddev dbeaver` inside wsl2

## How This PR Solves The Issue
Command dbeaver was modified to support dbeaver community installed to everyone

## Manual Testing Instructions
Install DBeaver Community edition in windows and call `ddev dbeaver` from wsl2.
It should open dbeaver and add the connection options to current project

## Related Issue Link(s)

* #5375
